test/grouping.jl

```diff
@@ -219,46 +219,57 @@ end
 
 @testset "@transform with grouped data frame" begin
 	d = DataFrame(n = 1:20, x = [3, 3, 3, 3, 1, 1, 1, 2, 1, 1, 2, 1, 1, 2, 2, 2, 3, 1, 1, 2])
-	g = groupby(d, :x, sort=true)
+	g = groupby(d, :x)
 
-	@test (@transform(g, y = :n .- median(:n)))[1,:y] == -5.0
+	@test (@transform(g, y = :n .- median(:n)))[1,:y] == -2.0
 
 	d = DataFrame(a = [1,1,1,2,2,3,3,1],
 	              b = Any[1,2,3,missing,missing,6.0,5.0,4],
 	              c = CategoricalArray([1,2,3,1,2,3,1,2]))
-      g = groupby(d, :a)
+    g = groupby(d, :a)
```

Comment:
Indentation seems to be incorrect here and in a few other places.

Reply:
indendation looks correct locally. I think github is displaying things weirdly.

Reply:
Looks like the rest of the file uses tabs. Better use tabs too, and fix the whole file later.

Reply:
Should be fixed now. Thanks for finding that.